README.md

```diff
@@ -81,36 +81,26 @@ $injector = $builder->build();
 $container = $injector->make(ContainerInterface::class);
 ```
 
-_**Note: An existing instance of Auryn can also be provided to the `build()` method.**_
+_**Note:** An existing instance of Auryn can also be provided to the `build()` method._
 
-### Service Definitions
+### Zend Service Manager Compatibility
 
-This package also includes a `ServiceConfig` class that supports applying a "service definition" map
-in the [Zend Service Manager][zend-service-manager] style:
+This package is compatible with [Zend Service Manager][zend-service-manager]:
 
 ```php
-use Northwoods\Container\InjectorBuilder;
-use Northwoods\Container\Config;
+use Northwoods\Container\Zend\Config;
+use Northwoods\Container\Zend\ContainerFactory;
 
-$builder = new InjectorBuilder([
-    new Config\ContainerConfig(),
-    new Config\ServiceConfig(require '/path/to/services.php'),
-]);
+$factory = new ContainerFactory();
 
-$injector = $builder->build();
+$container = $factory(new Config(
+    require 'path/to/services.php',
+));
 ```
 
-[zend-service-manager]: https://docs.zendframework.com/zend-servicemanager/configuring-the-service-manager/
-
-The following definitions are supported by `ServiceConfig`:
+_**Note:** All injections configured this way will be shared!_
 
-- `aliases` will be aliased
-- `delegators` will create a chained prepare
-- `factories` will create a delegate
-- `invokables` will be aliased
-- `services` will be wrapped as a delegate
-- `shared` will enable (or disable) sharing of specific classes
-- `shared_by_default` will enable (or disable) sharing by default
+[zend-service-manager]: https://docs.zendframework.com/zend-servicemanager/configuring-the-service-manager/
```

Comment:
I would suggest to add link to https://docs.zendframework.com/zend-expressive/v3/features/container/config/ instead of configuration of the ServiceManager. SM supports also shared/non-shared service, abstract_factories etc...

Reply:
Done.